react-time-track/src/components/add-member-project.js

```diff
@@ -5,8 +5,137 @@ import { FaAngleDown, FaAngleUp } from "react-icons/fa";
 
 import { Button } from "./ui";
 
+const sectionStyle = {
+  display: "flex",
+  flexDirection: "column",
+  alignItems: "center",
+  justifyContent: "center",
+  backgroundColor: "rgba(0,0,0,0.5)",
+  position: "fixed",
+  top: 0,
+  left: 0,
+  right: 0,
+  bottom: 0,
+  "@media (min-width: 960px)": {
+    left: 230
+  },
+  "@media (max-width: 450px)": {
+    justifyContent: "center"
+  }
+};
+
+const formStyle = {
+  width: "80%",
+  backgroundColor: "white",
+  padding: 10,
+  boxShadow: "0 20px 25px -5px rgba(0, 0, 0, 0.5)",
+  borderRadius: "0.5em"
+};
+
+const fieldsetStyle = {
+  width: "94%",
+  border: "none",
+  padding: "12px 30px",
+  textAlign: "center",
+  "@media (max-width: 800px)": {
+    padding: "12px 15px",
+    width: ""
+  },
+  "@media (max-width: 500px)": {
+    padding: "12px 8px",
+    width: ""
+  },
+  "@media (max-width: 450px)": {
+    padding: "16px 8px",
+    width: ""
+  }
+};
```

Comment:
Have you thought about following standardized measures for media queries?. Check this article: https://www.toptal.com/responsive-web/introduction-to-responsive-web-design-pseudo-elements-media-queries

Reply:
Good advice, I haven't thought about that, I am considering all the ways possible in how the application looks like for different medias, but the lecture could help me to learn about many things. Thanks